appium/webdriver/extensions/log_event.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from selenium import webdriver
+
+from ..mobilecommand import MobileCommand as Command
+
+
+class LogEvent(webdriver.Remote):
+
+    def get_events(self):
+        """ Retrieves events information from the current session
+        (Since Appium 1.16.0)
+
+        Usage:
+            events = driver.get_events()
+
+        Returns:
+            `dict`: A dictionary of events timing information containing the following entries
+                commands: (`list` of `dict`) List of dictionaries containing the following entries
+                    cmd: (str) The command name that has been sent to the appium server
+                    startTime: (int) Received time
+                    endTime: (init) Response time
+        """
+        return self.execute(Command.GET_EVENTS)['value']
```

Comment:
As a client, below two cases are available

`self.execute(Command.LOG_EVENT, {})`
`self.execute(Command.LOG_EVENT, {'type': nil})` (<= Appium server will handle this 'type' to filter events)

Reply:
Yes, I've checked and added, but removed by below commit since it isn't implemented yet.
In my opinion, it's enough to add `type` when it's ready.

https://github.com/appium/python-client/pull/469/commits/0a8406965cde76302cb1d19767d3722ad444de30

However, I'll restore `type` since users can use it as soon as server is ready, docstring is not ready though.

Reply:
It could depend on server-side if the parameter works.
We can handle the behaviour only server-side modification (and documentation on appium.io), if clients follow [the route definition](https://github.com/appium/appium-base-driver/blob/master/lib/protocol/routes.js) like other commands.